test/examples_test.go

```diff
@@ -23,6 +23,26 @@ func TestCoinPricesFromCoingecko(t *testing.T) {
 		RunScript(filepath.Join(scriptDir, projectName+".txtar"))
 }
 
+func TestHelloWorldOnChain(t *testing.T) {
+	projectName := "hello_world_on_chain"
+	projectDir := filepath.Join(examplesDir, projectName)
+	NewProjectTest(t, projectDir).
+		CopyDir("contracts/").
+		CopyDir("deployments/").
+		CopyDir("inputs/").
+		CopyDir("lib/").
+		CopyDir("scripts/").
+		CopyDir("test/").
+		CopyFile(".env").
+		CopyFile("hardhat.config.ts").
+		CopyFile("package.json").
+		CopyFile("package-lock.json").
+		CopyFile("tsconfig.json").
+		SetEnv("TEST_CASE", "Local").
```

Comment:
This env var can be used to switch between the local ("Local") and live ("Base Sepolia") tests in the `hello_world_on_chain/test/user.ts` test file

Reply:
Might be worth just hardcoding this for now and use a different script for "base sepolia" when later attempting to test live. There seems to be a trend towards keeping the testscripts as static as possible after setup (from what I've seen so far). Not necessarily something we need to do, but it might be worth slightly higher verbosity to have more clear-cut test conditions.